examples/example_logger.cpp

```diff
@@ -5,7 +5,7 @@ using yacx::loglevel;
 int main(int argc, char const *const *const argv) {
   yacx::handle_logging_args(argc, argv);
 
-  Logger(loglevel::ERROR) << "error message";
+  Logger(loglevel::ERR) << "error message";
```

Comment:
this sucks so bad, but msvc somewhere does this:
```c++
#define ERROR 0
```

Reply:
Guess I haven't yet changed it on the java side: https://github.com/ZerataX/yacx/pull/167/checks?check_run_id=659159835#step:3:218

Reply:
ich finde das ja sehr witzig, dass das schon vordefiniert ist